pkg/crc/manpages/manpages.go

```diff
@@ -0,0 +1,173 @@
+package manpages
```

Comment:
Since man pages is only useful for linux and mac so better to have the build target and because it is used for `root.go` so may be for windows just have a function which return nil?